ipatests/test_integration/test_ipahealthcheck.py

```diff
@@ -1227,6 +1227,29 @@ def test_sosreport_includes_healthcheck(self, create_logfile):
         )
         assert msg in cmd.stdout_text
 
+    def test_ipahealthcheck_verify_perms_for_source_files(self):
+        """
+        This tests checks if files in /var/log are checked with ipa.files
+        source.
+        The test modifies permissions of ipainstall log file and checks the
+        response from healthcheck.
+
+        https://pagure.io/freeipa/issue/8949
+        """
+        self.master.run_command(["chmod", "0644", paths.IPASERVER_INSTALL_LOG])
```

Comment:
We already have tests that check permissions and they are using the fixture `modify_permissions` (see for instance `test_tomcat_filecheck_too_restrictive`). For consistency it would be better to also use the fixture in this test.

Reply:
Thank you, Flo! Issue addressed.